libs/partners/qdrant/langchain_qdrant/vectorstores.py

```diff
@@ -1353,8 +1357,8 @@ def from_texts(
     def from_existing_collection(
         cls: Type[Qdrant],
         embedding: Embeddings,
-        path: str,
         collection_name: str,
+        path: Optional[str] = None,
```

Comment:
hm this is a breaking change (swapping the order of positional args). an alternative would be to give both path and collection_name defaults and just manually check that collection_name is passed in
```python
  path: Optional[str] = None,
  collection_name: Optional[str] = None,
  ...
  if collection_name is None:
    raise ValueError("Must specify collection_name, received None").
```